### ISSUE TITLE
Provide the documentation, or remove this sh*t from here. 

### ISSUE THREAD

**Issue by momole02** [bpmn-io/diagram-js#792]
Hi guys. 

 I wanted to use your library to manage my diagrams. But I don't see any documentation or API references. Nothing!

If you want to make a library that you're going to use only with your friends, make a private repository, and invite them on it, because the way it's done, it's really sucks. We who aren't in your sect can't use the library. 

I downloaded your example editor, I don't even know how to run it. 

In short, if you're not able to offer a minimum of documentation like other libraries do, you'd better remove your stuff from github because it's really not helping. 

And if we also have to pay licenses of 3,000 to 10,000 USD, like for your GoJS and JointJS buddies, to get quality diagrams and good documentation, that would be a real shame.

**Comment by nikku** [comment]
Thanks for your feedback @momole02. We apprechiate it.

If you need concrete advice on how to use diagram-js, reach out to us via [our forums](https://forum.bpmn.io/), checkout the [basic example](https://github.com/bpmn-io/diagram-js-examples) or dive deeper via actual libraries building on top of this tool.
